LukeMeilerHW8P1.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Wolfram Aplhoa still uses a numerical method I assume. Do you know which one?</p><p> </p><p>Eg Austin used numpy quad which is an old Fortran library: https://en.wikipedia.org/wiki/QUADPACK</p><p> </p><p>What boundaries did you use for Wolfram alpha for sin(1/x)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>